examples/constant/constant.bal

```diff
@@ -0,0 +1,36 @@
+import ballerina/io;
+
+// Constants are defined using the `const` modifier.
+const string GET = "GET";
+
+// Constants can be defined without type using `var` keyword. Then the type will be inferred from the RHS.
```

Comment:
Let's avoid future tense. Normally the word "will" is indicative of future tense in a sentence.  Change to simple present and say "is inferred.."

Shall we expand on what RHS is so that this is simpler for a newer user to follow?